README.md

```diff
@@ -81,12 +81,17 @@ There will be four meetings per year.   Three half day meetings (3 hours) will b
 
 **Solid Operations Advisory Group (formerly Solid team)** 
 
-This group advises on key areas including the solid website, hosting structure for PODs,  solidcommunity.net, github.
+In contrast, the Solid Operations Advisory Group is a volunteer group that is self-nominated and elected by the community. This group is not expected to do substantive day-to-day work, but rather act as representatives that guide key decisions - for instance, they would have been consulted on whether to proceed with the recent solidcommunity.net migration. They would also be advising on the infrastructure roadmap which would guide where employed maintenance staff in the Project/Initiative Team would focus their time.
 
-ODI is happy to help implement this group based on advice from the Advisory Committee. We would suggest that members are in place for a 2 year term. 
+Infrastructure that this group would advise on includes:
+ - solidproject.org
+ - solidcommunity.net
+ - Repositories under the [solid](https://github.com/solid/) and [solid-contrib](https://github.com/solid-contrib/) GitHub organisations
 
```

Comment:
Add a line for the ability of this group to recommend additions to this list as they deem fit.